somefile.txt

```diff
@@ -1,3 +1,8 @@
 hello Advait.
 how are you?
-i think that schroedinger is a bit dull. 
\ No newline at end of file
+i think that schroedinger is a bit dull. 
+
+Hello Galib.
+I am .. existing. 
+I agree that schoedinger is quite dull. 
+But you know he solved the hydrogen atom.
```

Comment:
Hmm. You have a point. But try giving him the _Uranium atom_. 

So could you please make some changes in the word `dull`?

## Todo
Just add three or four more `L`s at the end of `dull` and then commit and execute:
```
git push
```
Your PR will be automatically updated. And I will be happy to merge it. :neckbeard:

Reply:
Thanks for the feedback :)
Schroedinger is successfully more "dulllled".

Reply:
Cool! Now you can merge the pull request.
![image](https://user-images.githubusercontent.com/24398776/64916119-21d1fa00-d777-11e9-826f-67dd6de2f3e5.png)

choosing `squash and merge` will squash all the commits into one, and then place that commit on top of the latest commit in the commit history. So, do it! Commit message can also be changed during this process.

Reply:
This branch has no conflicts with the base branch
Only those with write access to this repository can merge pull requests...

Reply:
Cool! Merging this.